.storybook/preview.tsx

```diff
@@ -78,7 +78,6 @@ function makeViewports() {
 }
 
 const parameters = {
-	actions: { argTypesRegex: '^on[A-Z].*' },
```

Comment:
Interestingly, actions now work! 🎉 
<img width="635" alt="image" src="https://github.com/user-attachments/assets/edb3268a-7f9c-4a13-bc9f-5fbe0f479217">

Reply:
Interestingly, if you do remove this and use `action` like so `{ onClose: action('onClose') }`, then the action is logged twice (once as an array item). With this line, it's properly logged once 
<img width="610" alt="image" src="https://github.com/user-attachments/assets/ee115818-1dff-43da-b294-df44d2bd6be3">

Reply:
Is that dev mode side effect or something else?

https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#implicit-actions-can-not-be-used-during-rendering-for-example-in-the-play-function.

Reply:
I see. If it's deprecated, don't worry then!